composer.json

```diff
@@ -19,15 +19,20 @@
 		}
 	],
 	"require": {
-		"php": ">= 5.3",
+		"php": ">= 7.0",
```

Comment:
I think it's too soon to require 7.0 - I personally could no longer use this library if the requirement is 7.0, AWS Linux only officially supports 5.6 so that's what I'm stuck with for the time being.